### ISSUE TITLE
BreadcrumbTrail with enabled useHash and reload of the page

### ISSUE THREAD

**Issue by aviriel** [Alfresco/Aikau#918]
I'm not sure is it a bug report or feature request, but is it possible to make Breadcrumb trail to show the path to the current directory when reloading the page or returning back to the document library from the document page?

1. Simple configuration:
```
{
  name: "alfresco/documentlibrary/AlfBreadcrumbTrail",
  config: {
    useHash: true
  }
},
{
  name: "alfresco/documentlibrary/AlfDocumentList",
  config: {
    rootNode: "alfresco://user/home",
    rawData: true,
    useHash: true,
    widgets: [
      {
        name: "alfresco/documentlibrary/views/AlfSimpleView"
      }
    ]
  }
}

```
2. Go to some folder, the hash will be added to URL, breadcrumbtrail will be extended.
![selection_110](https://cloud.githubusercontent.com/assets/596948/13730771/06de0ab2-e96a-11e5-9ced-d91a4c636a5c.png)

3. Open this URL (reload the page). Document library parses the hash, breadcrumbtrail - no.
![selection_111](https://cloud.githubusercontent.com/assets/596948/13730775/404db70c-e96a-11e5-803b-b30a62243126.png)

4. Go to the next folder. Connection with breadcrumbtrail is reestablished.
![selection_113](https://cloud.githubusercontent.com/assets/596948/13730779/6578e420-e96a-11e5-9a9d-d5ef57bd72e4.png)

Looks like it should either work in all cases or don't work at all.

**Comment by draperd** [comment]
What version of Aikau is this on? It does sound like a bug.

**Comment by aviriel** [comment]
1.0.58, 1.0.39.5, also it's easily to reproduce it on https://github.com/draperd/MyAlfrescoFiles if add "useHash: true" to the breadcrumb.

**Comment by draperd** [comment]
Thanks for the update - I'll get a JIRA ticket raised for this

**Comment by draperd** [comment]
I've raised https://issues.alfresco.com/jira/browse/AKU-895 to cover this - as a bug it will be included in the next sprint. The next sprint is actually full loaded already (with nothing but bugs), but hopefully we should get through all the bugs and this will be in the 1.0.60 release.

**Comment by draperd** [comment]
The fix for this is now available in the 1.0.60 release.
